link-grammar/parse/prune.c

```diff
@@ -904,6 +904,25 @@ static bool check_null_word(prune_context *pc, int w)
 	return false;
 }
 
+/**
+ * Return \c true if a pass has not made any changes.
```

Comment:
stray backslash-c

Reply:
This is by intention, and I already used it several times.
It is a Doxygen escape for 'code'.
I also used several times `\p` (for parameter).
(These were the only Doxygen escapes I used by now.)